app/lib/context.rb

```diff
@@ -58,15 +58,11 @@ def steps
   end
 
   def clear
-    @form_context.clear(@form_id)
+    @form_context.clear(@form.id)
```

Comment:
Does the `attr_accessor` you've added mean you can use:
```suggestion
    @form_context.clear(form.id)
```